packages/d/devexpress-gantt.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "devexpress-gantt",
+  "filename": "dist/dx-gantt.min.js",
```

```suggestion
  "filename": "dx-gantt.min.js",
```
`dist` is the base path, so files are pulled from that into the root.